docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -72,6 +72,27 @@ public void publishMessage() {
 }
 ----
 
+==== Subscribing to a subscription
+
+Google Cloud Pub/Sub allows many subscriptions to be associated to the same topic.
+`PubSubTemplate` allows you to subscribe to subscriptions via the `subscribe()` method.
+It relies on a `SubscriberFactory` object, whose only task is to generate Google Cloud Pub/Sub
+`Subscriber`s.
```

Comment:
We have to add whitespace after "`" or move `s` into quotes.

AsciiDoc can't render the code properly without a whitespace symbol.